lib/cluster-providers/generic-cluster-provider.ts

```diff
@@ -3,11 +3,12 @@ import * as ec2 from "aws-cdk-lib/aws-ec2";
 import * as eks from "aws-cdk-lib/aws-eks";
 import { Construct } from "constructs";
 import { ClusterInfo, ClusterProvider } from "../spi";
-import { setPath, valueFromContext, Writeable } from "../utils";
+import { ArrayConstraint, ConstraintsType, NumberConstraint, setPath, valueFromContext, Writeable } from "../utils";
```

Comment:
The rule that we use: if the number of objects in the import is more than 3 (which is the case here: 6) you should use wildcard import e.g. `import * as utils from `